.github/workflows/continuous-integration-linux.yml

```diff
@@ -118,6 +118,7 @@ jobs:
             -DKokkos_ENABLE_DESUL_ATOMICS_EXTERNAL=ON \
             -DKokkos_ENABLE_HWLOC=ON \
             -DKokkos_ENABLE_${{ matrix.backend }}=ON \
+            -DKokkos_ENABLE_SERIAL=ON \
```

Comment:
Please drop that change
I'd rather have no coverage in CI that enabling the Serial backend in all these builds

Reply:
Dropped. That was my preference as well.